docs/guide/object-road.md

```diff
@@ -210,7 +210,7 @@ AddLaneRoadSegment {
 
 - `newLaneLeft`: Defines on which side is added the new lane.
 
-- `lines`: Defines the property of each line separating two lanes.
+- `lines`: Defines the property of each line separating two lanes. This field accepts the following value: `RoadLine {}`.
```

Comment:
This doesn't sound very clear to me. Rather I would say "This filed accepts a list of `RoadLine` nodes".